base/data/Campaign.js

```diff
@@ -47,6 +48,56 @@ Campaign.fetchFor = (advert,status=KStatus.DRAFT) => {
 	return pvc;
 };
 
+/**
+ * Get the master campaign of a multi campaign object
+ * @param {Advertiser|Agency} multiCampaign 
+ * @returns {Campaign}
+ */
+Campaign.fetchMasterCampaign = (multiCampaign, status=KStatus.DRAFT) => {
+    let pvCampaign = getDataItem({type:C.TYPES.Campaign,status,id:multiCampaign.campaign});
+    return pvCampaign && pvCampaign.value;
```

Comment:
It's usually better to return the pv, as that lets the caller distinguish between null vs loading.